dashboard/db/migrate/20161212222541_create_pd_teacher_applications.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePdTeacherApplications < ActiveRecord::Migration[5.0]
+  def change
+    create_table :pd_teacher_applications do |t|
+      t.timestamps
+
+      # Setting index false initially to create a unique index below. The unique param doesn't work here.
+      t.belongs_to :user, null: false, index: false
+      t.index :user_id, unique: true
+
+      t.string :primary_email, null: false, index: true
+      t.string :secondary_email, null: false, index: true
```

Comment:
Sort of. We are going to autopopulate their code studio email address into the primary field here. We will allow that, but we are associating their Code Studio ID with their application so we can sort it out down the line.